docs/source/configuration/metrics.mdx

```diff
@@ -56,11 +56,38 @@ apollo_router_http_request_duration_seconds_bucket{le="0.9"} 1
 
 > Note that if you haven't run a query against the router yet, you'll see a blank page because no metrics have been generated!
 
-### Available metrics
+## Datadog vas OTLP
+
+To use Datadog, you must configure the Datadog agent to accept OTLP metrics. This can be done by adding the following to your `datadog.yaml`:
+
+```yaml title="datadog.yaml"
+otlp_config:
+  receiver:
+    protocols:
+      grpc:
+        endpoint: localhost:4317
```

Comment:
Is this the expected syntax for datadog.yaml? This page uses both `127.0.0.1:4317` and `http://<datadog-agent>:4317` for the router.yaml. Should choose one (localhost or `datadog-agent`) and stick with it.

Reply:
Fixed in [bdb11b1](https://github.com/apollographql/router/pull/3962/commits/bdb11b1566c066a0ad39b9cf2fc51b5ccb9ce03c)